### DISCUSSION TITLE
cpio 2GB archive limitation

### DISCUSSION THREAD

**Discussion by lfdmn** [OE4T/meta-tegra#534] [Q&A] [answered]
Hi,

We use cpio archive for tegra-minimal-initramfs and it is required.

How can I disable the cpio archive creation for the rootfs image? I get this error:

```
ERROR: image-1.0-r0 do_image_cpio: Execution of '.../build-jetson-xavier/tmp/work/jetson_xavier-poky-linux/image/1.0-r0/temp/run.do_image_cpio.14552' failed with exit code 2:
4397300 blocks
cpio: cannot seek on output: Invalid argument
```
now that I reach the 2G cpio limitation.

**Comment by madisongh** [comment, marked as answer]
The IMAGE_FSTYPES variable controls the types of images that get created. Not sure why you'd have cpio in the list for a non-initramfs image... you might want to to track down where it's getting added and fix that.  Alternatively, you could just remove cpio (plus any variants, like cpio.gz) from that variable in your image recipe.

**Comment by lfdmn** [comment]
OK thanks @madisongh . 

I did forget to set IMAGE_FSTYPES with my custom image type in the image recipe.
